src/utils/BlockUtils.ts

```diff
@@ -0,0 +1,53 @@
+import { BlockStruct } from "@typechain-types/contracts/V1/DataTypes";
+import { AddressLike } from "ethers";
+
+/**
+ * Extract numeric fields from a block and convert them to regular number types
+ */
+export const coordinatesOf = (block: BlockStruct) => ({
+    forkCnt: Number(block.transaction.header.forkCnt),
+    height: Number(block.transaction.header.transactionCnt)
+});
+
+/**
+ * Get the block height (transaction count) from a block
+ */
+export const heightOf = (block: BlockStruct): number =>
```

Comment:
Maybe better to name everything with 'get' instead of 'of' since we use 'get' across the codebase -> getHeight

Reply:
i like the `of` because it reads nicely
`heightOf(block)`

but you're the boss. if you prefer `get` then `get` it will be